tools/uploader-function/src/DicomUploaderFunction/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Azure Functions localsettings file
+local.settings.json
```

Comment:
Is this file necessary? I think we already ignore this

Reply:
Doesn't look like we do, but I'll add it to the global `.gitignore`